### PR TITLE
Android switcher color

### PR DESCRIPTION
**Pull request by hawkrives**
Closes https://github.com/StoDevX/AAO-React-Native/issues/2580

Also, we changed the accent color for check boxes and switches:

It's now purple, from the list of approved St. Olaf colors.

### CODE REVIEW

**Comment by drewvolz** [comment]
...purple?

**Comment by drewvolz** [comment]
I don't know that our color scheme of gold, black, and purple looks the best. We should highly reconsider.

**Comment by rye** [comment]
Purple doesn't seem that good in context.  It looked good on the theme editor, but kinda clashes with the gold title bar.

**Comment by hawkrives** [comment]
I am open to suggestions.

**Comment by rye** [comment]
I think I might like a dark grey or black, potentially.

How does Chapel Cranberry look on this?

File: android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Change the color value for your need -->
+    <color name="colorPrimary">#ffc107</color>
+    <color name="colorPrimaryDark">#c79100</color>
+    <color name="colorAccent">#6E3A5D</color>
```
Comment (rye) [inline android/app/src/main/res/values/colors.xml:6]: This should be downcased.

File: android/build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0-alpha09'
+        classpath 'com.android.tools.build:gradle:3.1.2'
```
Comment (rye) [inline android/build.gradle:9]: Why was this change necessary?  (Or was this automatically performed?)

**Review by rye — COMMENTED** [review]
I was thinking about #f98e2c as a possible accent color, too.  (Mellby Orange)

Or Chapel Cranberry. (#ac145a-ish)

**Comment by rye** [comment]
The more I think about it, though, purple is actually a pretty cool color for these cases… it's roughly opposite the color wheel from yellow/gold.  I wish we had more screenshots. 😛

File: android/build.gradle (hunk 2 above)
Comment (hawkrives) [reply android/build.gradle:9]: I think Android Studio was yelling at me / did it automatically

**Comment by hawkrives** [comment]
I still like purple, as well 🐉

My android emulator isn't working atm, so I can't do more screenshots.
